tests/acceptance/bootstrap/SharingNgContext.php

```diff
@@ -1878,16 +1886,7 @@ public function userTriesToListThePermissionsOfSpaceOwnedByUsingRootEndpointOfTh
 		string $space,
 		string $spaceOwner
 	): void {
-		$spaceId = ($this->spacesContext->getSpaceByName($spaceOwner, $space))["id"];
-
-		$response = GraphHelper::getDrivePermissionsList(
-			$this->featureContext->getBaseUrl(),
-			$this->featureContext->getStepLineRef(),
-			$user,
-			$this->featureContext->getPasswordForUser($user),
-			$spaceId
-		);
-		$this->featureContext->setResponse($response);
+		$this->featureContext->setResponse($this->getDrivePermissionsList($user, $space));
```

Comment:
We also need to pass `$spaceOwner` to `getDrivePermissionsList()` because in this step the space owner is not the user.
That is also the reason why CI is failing

Reply:
yeah, true. Fixed 👍